sce.sh

```diff
@@ -225,12 +187,12 @@ then
     then
         print_missing_config $REPO_LOCATION $missingPaths
     fi
-    if [ $start_only_mongodb_container -eq 0 ]
+    if [ $start_only_mongodb_container == 0 ]
```

Comment:
lets make it so 0 means false, and 1 means true

if `start_only_mongodb_container` is 0, dont do it, if it is 1, start mongo only

Reply:
I wanted to be consistent with whoever wrote it before and they used 0 to indicate a successful exit (in this case, yes for starting container). Do you still want me to update it?

Reply:
exit codes are like "did something bad happen?" 0 means no, any other code means yes, the code value being an indicator of what went wrong

in both cases, 0 means false, pls update when you can

Reply:
done